File: kijong/12_14/pg_42584/42584.py
def solution(prices):
    answer =[]
    for i in range(len(prices)):
        count = 0
        for j in range(i+1, len(prices)):
            count += 1 
            if (prices[i] <= prices[j]):
                continue
            else: 
                break
        answer.append(count)
    #answer.append(0)
    return answer

File: kijong/12_14/pg_42584/42584.cpp
#include <string>
#include <iostream>
#include <vector>

using namespace std;

vector<int> solution(vector<int> prices) {
    vector<int> answer;
    for(int i=0; i<prices.size(); i++){
        int count = 0;
        for(int j=i+1; j<prices.size();j++){
            count++;
            if(prices[i] <= prices[j] ){
                continue;
                }
            else{
                break;
            }
            }
        answer.push_back(count);
        }
    return answer;
}